src/compile_torture_tests.py

```diff
@@ -57,7 +57,7 @@ def find_runnable_tests(directory, pattern):
     for filename in files:
       if fnmatch.fnmatch(filename, pattern):
         fullname = os.path.join(root, filename)
-        with open(fullname, 'r') as f:
+        with open(fullname, 'r', encoding='ISO8859-1') as f:
```

Comment:
wat?

Reply:
Apparently it makes a difference. By default it tries UTF-8 encoding and fails to read several of the tests (the offending characters appear to be in comments)

Reply:
Holy crap.   Can you add a command about this?